dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityPreviewMode.java

```diff
@@ -0,0 +1,90 @@
+package com.dotcms.rendering.velocity.servlet;
+
+import com.dotcms.rendering.velocity.services.PageContextBuilder;
+import com.dotcms.rendering.velocity.services.VelocityType;
+import com.dotcms.rendering.velocity.util.VelocityUtil;
+import com.dotcms.rendering.velocity.viewtools.content.ContentMap;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.IHTMLPage;
+import com.dotmarketing.util.PageMode;
+
+import java.io.File;
+import java.io.Writer;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.velocity.context.Context;
+
+import com.liferay.portal.model.User;
+
+public class VelocityPreviewMode extends VelocityModeHandler {
+
+
+
+    private final HttpServletRequest request;
+    private final HttpServletResponse response;
+    private final PageMode mode = PageMode.PREVIEW_MODE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)